acceptance/suites/tests/00_smoke/master_starts_if_agent_run_before_ssl_files_inited.rb

```diff
@@ -7,6 +7,10 @@
 puppetservice=options['puppetservice']
 ssldir = master.puppet['ssldir']
 backup_ssldir = master.tmpdir("agent_run_before_master_init_ssldir_backup")
+reports_termini = on(master, puppet("config print --section master reports")).stdout.chomp
+puppetdb_enabled = reports_termini =~ /puppetdb/
+route_file_exists = on(master, "ls /etc/puppetlabs/puppet/").stdout =~ /routes\.yaml/
```

Comment:
nit: I think `test -f /etc/puppetlabs/puppet/routes.yaml` would be better here, but :woman_shrugging: